legal-api/src/legal_api/services/authz.py

```diff
@@ -88,3 +90,100 @@ def has_roles(jwt: JwtManager, roles: List[str]) -> bool:
     if jwt.validate_roles(roles):
         return True
     return False
+
+
+ALLOWABLE_FILINGS_ACTIVE: Final = {
+    'alteration': {
+        'staff': ['BC', 'BEN'],
+        'user': ['BC', 'BEN'],
+    },
+    'annualReport': {
+        'staff': ['CP', 'BEN'],
+        'user': ['CP', 'BEN'],
+    },
+    'changeOfAddress': {
+        'staff': ['CP', 'BEN'],
+        'user': ['CP', 'BEN'],
+    },
+    'changeOfDirectors': {
+        'staff': ['CP', 'BEN'],
+        'user': ['CP', 'BEN'],
+    },
+    'correction': {
+        'staff': ['CP', 'BEN'],
+    },
+    'courtOrder': {
+        'staff': ['CP', 'BC', 'BEN', 'CC', 'ULC', 'LLC'],
+    },
+    'dissolution': {
+        'staff': ['CP', 'BC', 'BEN', 'CC', 'ULC', 'LLC'],
+        'user': ['CP', 'BC', 'BEN', 'CC', 'ULC', 'LLC'],
+    },
+    'incorporationApplication': {
+        'staff': ['CP', 'BC', 'BEN'],
+        'user': ['CP', 'BC', 'BEN'],
+    },
+    'restoration': {
+        'staff': ['CP', 'BC', 'BEN', 'CC', 'ULC', 'LLC'],
+    },
+    'specialResolution': {
+        'staff': ['CP'],
+        'user': ['CP'],
+    },
+    'transition': {
+        'staff': ['BC', 'BEN'],
+        'user': ['BC', 'BEN'],
+    },
+    'registrarsNotation': {
+        'staff': ['CP', 'BC', 'BEN', 'CC', 'ULC', 'LLC'],
+    },
+    'registrarsOrder': {
+        'staff': ['CP', 'BC', 'BEN', 'CC', 'ULC', 'LLC'],
+    },
+}
+
+ALLOWABLE_FILINGS_HISTORICAL: Final = {
+    'courtOrder': {
+        'staff': ['CP', 'BC', 'BEN', 'CC', 'ULC', 'LLC'],
+    },
+    'registrarsNotation': {
+        'staff': ['CP', 'BC', 'BEN', 'CC', 'ULC', 'LLC'],
+    },
+    'registrarsOrder': {
+        'staff': ['CP', 'BC', 'BEN', 'CC', 'ULC', 'LLC'],
+    },
+    'restoration': {
+        'staff': ['CP', 'BC', 'BEN', 'CC', 'ULC', 'LLC'],
+    },
+}
```

Comment:
Please look at the kinds of actions that the UI currently "allows"... Will the API be able to provide True/False for a similar list?

https://github.com/bcgov/business-filings-ui/blob/main/src/mixins/allowable-actions-mixin.ts